alpha/scripts/analyzeEmailFieldForPartner.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+require_once(__DIR__ . '/bootstrap.php');
+
+// parse the command line
+if($argc<2)
+{
+	die("Usage: php $argv[0] <partner id>");
+}
+
+$partnerId = $argv[1];
+$userType = $argv[2];
+$dryRun = $argv[3];
+var_dump($partnerId, $userType, $dryRun);
+
+if (!PartnerPeer::retrieveByPK($partnerId))
+{
+	die("Please enter a valid partner Id!\n");
+}
+
+if (!$userType || ($userType != "admin" && $userType != "user"))
+{
+	die("Please specify if looking for admin users (admin) or regular users (user)\n");
+}
+
+if (!$dryRun || ($dryRun != "dryRun" && $dryRun != "wetRun"))
```

Comment:
realRun is the common term for the oposite of dryRun
also just check if what you got is not realRun -> just set dryrUN